provider/openstack/firewaller.go

```diff
@@ -480,7 +688,6 @@ func (c *defaultFirewaller) closePortsInGroup(nameRegExp string, portRanges []ne
 		return err
 	}
 	neutronClient := c.environ.neutron()
-	// TODO: Hey look ma, it's quadratic
```

Comment:
Honestly it matter much les since the introduction of port range support, as the only time normal use would really get hundreds of rules was through that (and would often run into quota issues anyway). It's fine to go (or at least get better wording).